examples/vision/mixUp.py

```diff
@@ -195,25 +198,21 @@ def get_training_model():
 """
 Readers are encouraged to try out mixup on different datasets from different domains and
 experiment with the $\lambda$ parameter. You are strongly advised to check out the
-original paper as well. In it, the authors present several ablation studies on mixup
-showing how it can improve generalization. Along with that, they also present results
-when you combine more than two images to create a single one.
+[original paper](https://arxiv.org/abs/1710.09412) as well - the authors present several ablation studies on mixup
+showing how it can improve generalization, as well as show their results of combining
+more than two images to create a single one.
 """
 
 """
 ## Notes
 
-* mixup can be extremely useful for low data regimes.
-* Label-smoothing and mixup usually do not complement each other.
+* mixup can be useful for low data regimes as it can help to create synthentic examples
+in a simple manner without incurring excruciating computational costs.
```

Comment:
Suggestion: refactor

```suggestion
* With mixup, you can create synthetic examples—especially when you lack a large dataset— 
without incurring high computational costs.
```

("simple" is relative 🤷‍♂️  "excruciating" -> "high"; anyway, just a suggestion)